compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -569,6 +569,23 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             _ => None,
         };
 
+        if intrinsic == Some(sym::const_eval_select) {
```

Comment:
The cranelift backend probably needs something similar (cc @bjorn3)

Reply:
Indeed. That did be in rustc_codegen_cranelift/src/intrinsics/mod.rs. You can call `abi::codegen_terminator_call` for codegening the actual call. Make sure that you return from the intrinsic handling function directly after this call to avoid two jump instructions after each other in the same basic block.